sourcegraph/sourcegraph.proto

```diff
@@ -288,58 +289,115 @@ message RepoList {
 
 service RepoBadges {
 	// ListBadges lists the available badges for repo.
-	rpc ListBadges(RepoSpec) returns (BadgeList);
+	rpc ListBadges(RepoSpec) returns (BadgeList) {
+		option (google.api.http) = {
+			get: "/v1/repo_badges/list_badges"
```

Comment:
Another question to discuss. You went with placing API version in the path, ala `/{version}/foo/bar`.

Do we think that's the way to go? An alternative would be having the version specified (optionally) via HTTP headers. That keeps the version more hidden and the paths a little shorter.

What do we think about this?